contributors.js

```diff
@@ -40,4 +40,5 @@ const contributors = [
   { username: 'Dhruv-Davda', message:'Good Wishes Everyone' },
   { username: 'Piyush-Goenka', message:'October is hacked' },
   { username: 'MeuHubPython', message: 'My duck is the best! Happy Hacktoberfest!' },
+  { username: 'xnisha-verma', message: 'Cheers to Hacktoberfest'24 :)' },
```

Comment:
Hello, thanks for your contribution. be careful, the json file is not correct. Please remove the ` or escape it and remove the comma at the end of the line 🙂

Reply:
Thank you for pointing out the issue. I've made necessary changes in json file.